README.md

```diff
@@ -57,50 +89,85 @@ print(posterior)
-Jupyter notebooks.
+You can also access and run these tutorials directly in your browser by opening
+[Codespace](https://docs.github.com/en/codespaces/overview). To do so, click the green
+“Code” button on the GitHub repository and select “Open with Codespaces.” This provides
+a fully functional environment where you can explore sbi through Jupyter notebooks.
 
 ## Inference Algorithms
 
-The following inference algorithms are currently available. You can find instructions on how to run each of these methods [here](https://sbi-dev.github.io/sbi/latest/tutorial/16_implemented_methods/).
+The following inference algorithms are currently available. You can find instructions on
+how to run each of these methods
+[here](https://sbi-dev.github.io/sbi/latest/tutorials/16_implemented_methods/).
 
 ### Neural Posterior Estimation: amortized (NPE) and sequential (SNPE)
 
-* [`SNPE_A`](https://sbi-dev.github.io/sbi/reference/#sbi.inference.snpe.snpe_a.SNPE_A) (including amortized single-round `NPE`) from Papamakarios G and Murray I [_Fast ε-free Inference of Simulation Models with Bayesian Conditional Density Estimation_](https://proceedings.neurips.cc/paper/2016/hash/6aca97005c68f1206823815f66102863-Abstract.html) (NeurIPS 2016).
+* [`SNPE_A`](https://sbi-dev.github.io/sbi/reference/#sbi.inference.snpe.snpe_a.SNPE_A)
+  (including amortized single-round `NPE`) from Papamakarios G and Murray I [_Fast
+  ε-free Inference of Simulation Models with Bayesian Conditional Density
+  Estimation_](https://proceedings.neurips.cc/paper/2016/hash/6aca97005c68f1206823815f66102863-Abstract.html)
+  (NeurIPS 2016).
+
+* [`SNPE_C`](https://sbi-dev.github.io/sbi/reference/#sbi.inference.snpe.snpe_c.SNPE_C)
+  or `APT` from Greenberg D, Nonnenmacher M, and Macke J [_Automatic Posterior
+  Transformation for likelihood-free inference_](https://arxiv.org/abs/1905.07488) (ICML
+  2019).
+
+* `TSNPE` from Deistler M, Goncalves P, and Macke J [_Truncated proposals for scalable
+  and hassle-free simulation-based inference_](https://arxiv.org/abs/2210.04815)
+  (NeurIPS 2022).
 
-* [`SNPE_C`](https://sbi-dev.github.io/sbi/reference/#sbi.inference.snpe.snpe_c.SNPE_C) or `APT` from Greenberg D, Nonnenmacher M, and Macke J [_Automatic
-  Posterior Transformation for likelihood-free
-  inference_](https://arxiv.org/abs/1905.07488) (ICML 2019).
+* [`FMPE`](https://sbi-dev.github.io/sbi/reference/#sbi.inference.fmpe.fmpe_base.FMPE)
+  from Wildberger, J., Dax, M., Buchholz, S., Green, S., Macke, J. H., & Schölkopf, B.
+  [_Flow matching for scalable simulation-based inference_]
+  (https://proceedings.neurips.cc/paper_files/paper/2023/hash/3663ae53ec078860bb0b9c6606e092a0-Abstract-Conference.html).(NeurIPS
+  2023).
 
-* `TSNPE` from Deistler M, Goncalves P, and Macke J [_Truncated proposals for scalable and hassle-free simulation-based inference_](https://arxiv.org/abs/2210.04815) (NeurIPS 2022).
+* [`NPSE`](https://sbi-dev.github.io/sbi/reference/#sbi.inference.npse.npse.NPSE) from
```

Comment:
isn't the better citation the Sharrock paper?

Reply:
Not sure, if we are just doing single round it's the same, no? and I think Geffner was first?

Reply:
Sharrock was first, and Geffner has focus on IID inference. So we will use the former for now and add the latter when we implement IID bridges.